elastic/kibana
# [Security Solution][Alert details] Fix analyzer preview navigation to timeline

## Summary

This PR fixed the analyzer preview (in alerts flyout) to timeline navigation. The user is now directed to the `analyzer` tab not `query` tab. Same to session view, if session view is available, clicking the session preview title should open timeline and go to `Session view` tab.

Related
- https://github.com/elastic/kibana/issues/180667
- https://github.com/elastic/kibana/issues/181232
- https://github.com/elastic/kibana/issues/169623 related to rule with suppression
- https://github.com/elastic/kibana/issues/166052 related to rule with timeline template

How to test
- Generate some alerts with analyzer and session view data
- Expand a row in alerts table
- Scroll down alerts detail flyout and go to `Visualization` section
- Click `Analyzer preview` should open timeline and go to `Analyzer` tab
- Click `Session viewer preview` should open timeline and go to `Session view` tab (enterprise license)

Basic scenario (Alert details -> Click Analyzer preview -> Timeline)

https://github.com/elastic/kibana/assets/18648970/ff617c07-d269-4108-8a3d-3744872ee66d

Rule with suppression & timeline template


https://github.com/elastic/kibana/assets/18648970/242d545c-d871-4c42-9ec8-5aa2fcd783ba
